package.json

```diff
@@ -23,7 +23,7 @@
     "@bcrs-shared-components/base-address": "2.0.3",
     "@bcrs-shared-components/breadcrumb": "2.1.5",
     "@bcrs-shared-components/business-lookup": "1.3.4",
-    "@bcrs-shared-components/certify": "2.1.5",
+    "@bcrs-shared-components/certify": "^2.1.55",
```

Comment:
Remove the `^` so that we import that exact version. A future version might have some incompatible changes.

Reply:
Thanks, done